src/Lesson1/Task3.js

```diff
@@ -0,0 +1,130 @@
+// 3. Необходимо написать иерархию классов вида:
+
+// Human -> Employee -> Developer
+// Human -> Employee -> Manager
+
+// Каждый Менеджер (Manager) должен иметь внутренний массив своих сотрудников (разработчиков), 
+// а также методы по удалению/добавлению разработчиков.
+// Каждый Разработчик (Developer) должны иметь ссылку на Менеджера и методы для изменения менеджера 
+// (имеется ввиду возможность назначить другого менеджера).
+// У класса Human должны быть следующие параметры: name (строка), age (число), dateOfBirth (строка или дата)
+// У класса Employee должны присутствовать параметры: salary (число), department (строка)
+// В классе Human должен присутствовать метод displayInfo, который возвращает строку со всеми параметрами экземпляра Human.
+// В классе Employee должен быть реализовать такой же метод (displayInfo), 
+// который вызывает базовый метод и дополняет его параметрами из экземпляра Employee
+// Чтобы вызвать метод базового класса, необходимо внутри вызова метода displayInfo класса Employee написать: 
+// super.displayInfo(), это вызовет метод disaplyInfo класс Human и вернет строку с параметрами Human'a.
+
+class Human {
+    constructor(name, age, dateOfBirth) {
+        this.name = name;
+        this.age = age;
+        this.dateOfBirth = dateOfBirth;
+    }
+
+    displayInfo() { return ['Имя:', this.name, 'Возраст:', this.age, 'Дата рождения:', this.dateOfBirth.toLocaleDateString()].join(' '); }
+}
+
+class Employee extends Human {
+    constructor(name, age, dateOfBirth, salary, department) {
+        super(name, age, dateOfBirth);
+        this.salary = salary;
+        this.department = department;
+        this.position=undefined;
+    }
+
+    displayInfo() {
+        return super.displayInfo() +' '+ ['Зарплата:', this.salary, 'Отдел:', this.department,'Должность:',this.position].join(' ');
+    }
+}
+
+class Developer extends Employee {
+    constructor(name, age, dateOfBirth, salary, department,manager=undefined){
```

Comment:
```suggestion
    constructor(name, age, dateOfBirth, salary, department,manager){
```
все равно внутри будет undefined, если мы ничего не передадим:)